docs/platform/custom-cloud-credentials/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+# Bring Your Own Cluster (BYOC) Troubleshooting
+
+## Deploying BYOC
+
+### Kubernetes API Authentication Errors
+AWS STS regional endpoints have to be enabled in the target region. Go to https://console.aws.amazon.com/iam/home#/account_settings and verify the regional endpoint is activated. In most cases your region already has AWS STS regional endpoint enabled -> https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_enable-regions.html
+
+!!! note
+    Skipping this step will cause difficult to debug issues. The kubelet will be unable to authenticate against the kubernetes API server, and nothing will work.
+
+## Deploying Workloads 
+
+### Stuck in Pending State
+There are usually three scenarios for why a session or run will be stuck in a pending state: awaiting AWS provisioning, one time start-up process **(sessions only)**, or the instance type is not available in the availability zones your BYOC cluster has access to. We will cover each of the three scenarios in detail here.
```

Comment:
Use numbered list when listing things numerically. Capitalize Session and Run. 

There are usually three scenarios for why a Session or Run will be stuck in a pending state: 
1. Awaiting AWS provisioning
2. For Sessions only: Initial start-up process (only happens once)
3. The requested Instance type is not available in the availability zones your cluster has access to

We will cover each of the three scenarios in detail here.